lumicks/pylake/force_calibration/tests/test_power_spectrum_calibration.py

```diff
@@ -97,6 +98,58 @@ def test_good_fit_integration_test(
     np.testing.assert_allclose(ps_calibration["err_alpha"], err_alpha, rtol=1e-6)
 
 
+def test_fit_settings(reference_models):
```

Comment:
Can you add a docstring that very briefly summarizes what this test is ensuring? Not what it's doing, just what its purpose is.

Reply:
Will do!